src/app/components/test-card-detailed/test-card-detailed.component.ts

```diff
@@ -180,4 +197,7 @@ export class TestCardDetailedComponent {
 
   }
 
+  goToLink(url: string) {
+    window.open(url, "_blank");
```

Comment:
is it link to some external resource?

Reply:
yes,  it's a link to external resource of selected film (facebook, instagram, twitter, homeFilmPage)

Reply:
change customer-feature-a to my-feature-modules and components with letter F: CardDetailedFComponent + GenreFComponent  + MovieCardFComponent + ResultsFComponent + TopBilledCastCardFComponent